xbmc/cores/AudioEngine/Engines/ActiveAE/ActiveAESound.h

```diff
@@ -59,6 +61,7 @@ class CActiveAESound : public IAESound
   static int64_t Seek(void *h, int64_t pos, int whence);
 
 protected:
+  CActiveAE *m_activeAE;
```

Comment:
(almost) everybody can comment but the maintainer decides what is acceped and what is recected.